pkg/auth/jwtverifier.go

```diff
@@ -72,7 +77,12 @@ func (v *JWTVerifier) Verify(tokenString string) (*Token, error) {
 	token, err := jwt.ParseWithClaims(
 		tokenString,
 		claims,
-		func(token *jwt.Token) (interface{}, error) {
+		func(token *jwt.Token) (any, error) {
+			// keyFunc (JWKS) takes precedence over the other ways of verification.
```

Comment:
I thought your proposal was to use JWKS if the token contained `kid`? Just curious why you decided against that (I don't have an option either way so happy with what you suggest)

Reply:
sure sounds good